Octokit/Clients/ReactionsClient.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Octokit
+{
+    public class ReactionsClient : ApiClient, IReactionsClient
+    {
+        /// <summary>
+        /// Instantiates a new GitHub Reactions API client
+        /// </summary>
+        /// <param name="apiConnection">An API connection</param>
+        public ReactionsClient(IApiConnection apiConnection)
+            : base(apiConnection)
+        {
+            CommitComments = new CommitCommentReactionClient(apiConnection);
```

Comment:
we normally use a singular form for the variable name and a plural form for the class.  so in this case

`CommitComment = new CommitCommentReactionsClient()`